litmus-portal/Makefile

```diff
@@ -68,4 +68,22 @@ backend-services-checks:
 		echo "go mod tidy checking failed!" && echo "$${tidyRes}" \
 		&& echo "Please ensure you are using $$($(GO) version) for formatting code." \
 		&& exit 1; \
-	fi
\ No newline at end of file
+    fi
```

Comment:
script is not indent

Reply:
I don't think there's any problem in the script, I ran the make file it works. If there was an indentation error it wouldn't work. Locally I can run the makefile : make backend-services-checks

```
------------------
--> checking code style [backend]
------------------
------------------
--> Check litmus-portal backend [go mod tidy]
------------------
------------------
--> Check litmus-portal auth [go mod tidy]
------------------
------------------
--> Check litmus-portal self-deployer [go mod tidy]
------------------
------------------
--> Check litmus-portal workflow-agent [go mod tidy]
------------------
```

Reply:
Even in the CI all the checks ran correctly

Reply:
![Selection_029](https://user-images.githubusercontent.com/20414564/92991792-cb230b80-f503-11ea-9198-5304ed6e76be.png)
Check the script, it is not indented properly. Edit it from the GitHub GUI.

Reply:
It is a github problem, on my system the indentation is correct. I have replace all the 4 spaced tabs with 8 spaces, it should solve the problem

Reply:
Can you edit it from GitHub IDE?